extensions-contrib/kafka-emitter/src/test/java/org/apache/druid/emitter/kafka/KafkaEmitterConfigTest.java

```diff
@@ -83,6 +84,17 @@ public void testSerDeNotRequiredKafkaProducerConfig()
     }
   }
 
+  @Test
+  public void testDeserializeEventTypesWithDifferentCase() throws JsonProcessingException
+  {
+    Assert.assertEquals(KafkaEmitterConfig.EventType.SEGMENTMETADATA, mapper.readValue("\"SEGMENTMETADATA\"", KafkaEmitterConfig.EventType.class));
+    Assert.assertEquals(KafkaEmitterConfig.EventType.SEGMENTMETADATA, mapper.readValue("\"segmentMetadata\"", KafkaEmitterConfig.EventType.class));
```

Comment:
this one should be enough, we don't need to support other casing, since those are not explicitly documented

Reply:
Updated!